about.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="Twee-Fi Milestone 1">
-    <meta name="author" content=""> 
+    <meta name="author" content="">
```

Comment:
What is this good for?

Reply:
Is done automagically by my IDE that removes unwanted spaces